.circleci/config.yml

```diff
@@ -26,23 +26,23 @@ commands:
       - checkout
       - run:
           name: Install Dependencies
-          command: npm ci
+          command: npm ci && npm run bootstrap
       - run:
           name: Run Linter
           command: npm run lint
       - run:
           name: Run Tests
-          command: npm t
+          command: npm run test
 
   test-built-app:
     steps:
       - checkout
       - run:
           name: Install Dependencies
-          command: npm ci
+          command: npm ci && npm run bootstrap
       - run:
           name: Link package
-          command: npm link
+          command: cd packages/create-contentful-app && npm link
```

Comment:
There is `working_directory` in Circle CI: https://circleci.com/docs/2.0/configuration-reference/#run

Reply:
Thanks for the catch -.- It's a leftover